docs/antipatterns/chatty-io/index.md

```diff
@@ -12,7 +12,7 @@ ms.custom: seodec18
 
 # Chatty I/O antipattern
 
-The cumulative effect of a large number of I/O requests can have a significant impact on performance and responsiveness.
+The cumulative effect of a large number of I/O requests can have a significant effect on performance and responsiveness.
```

Comment:
I think 'impact' is OK here. Effect is any result of a cause, impact implies a negative effect which is the case here. (We should avoid 'impact' as a verb however)

Reply:
Reverted